src/app/dataset-view/additional-components/overview-component/components/readme-section/readme-section.component.ts

```diff
@@ -0,0 +1,86 @@
+import {
+    ChangeDetectionStrategy,
+    Component,
+    Input,
+    OnInit,
+} from "@angular/core";
+import {
+    DatasetBasicsFragment,
+    DatasetDataSizeFragment,
+    DatasetOverviewFragment,
+} from "src/app/api/kamu.graphql.interface";
+import { MaybeNull } from "src/app/common/app.types";
+import { BaseComponent } from "src/app/common/base.component";
+import { AppDatasetCreateService } from "src/app/dataset-create/dataset-create.service";
+import { OverviewDataUpdate } from "src/app/dataset-view/dataset.subscriptions.interface";
+import { AppDatasetSubscriptionsService } from "src/app/dataset-view/dataset.subscriptions.service";
+import { DatasetSchema, DataRow } from "src/app/interface/dataset.interface";
+
+@Component({
+    selector: "app-readme-section",
+    templateUrl: "./readme-section.component.html",
+    styleUrls: ["./readme-section.component.sass"],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ReadmeSectionComponent extends BaseComponent implements OnInit {
+    @Input() public datasetBasics?: DatasetBasicsFragment;
+    @Input() public currentState?: {
+        schema: MaybeNull<DatasetSchema>;
+        data: DataRow[];
+        overview: DatasetOverviewFragment;
+        size: DatasetDataSizeFragment;
+    };
+    public isEditMode = true;
```

Comment:
Make an enum to distinguish Edit and Preview tabs. Boolean si totally not obvious.

Reply:
Add enum EditMode